src/triton_interfaces/msg/DetectionBox.msg

```diff
@@ -0,0 +1,6 @@
+float32 x
```

Comment:
Be sure to leave a short comment explaining the message parameters for any custom messages. For example I am unsure if x,y is the centre of the bounding box (most likely) or a corner of it.